### PR TITLE
[bugfix] Review status count error fix

### PR DESCRIPTION
**Pull request by bruntib**
The SQL statement querying review status counts when filtering reports
in GUI with uniqueing was invalid.

### CODE REVIEW

**Review by vodorok — CHANGES_REQUESTED** [review]
I would rephrase the commit message:
The SQL statement querying review status counts was invalid, when filtering reports in rhe GUI with uniqueing enabled.

Otherwise LGTM. Good catch!  :rocket:

**Review by dkrupp — APPROVED** [review]

**Review by vodorok — APPROVED** [review]
